mcnode/ds.go

```diff
@@ -105,6 +108,42 @@ func (ds *RocksDS) Delete(key Key) error {
 	return ds.db.Delete(ds.wo, key[2:])
 }
 
+func (ds *RocksDS) Sync() error {
+	return ds.db.Flush(ds.fo)
+}
+
+func (ds *RocksDS) IterKeys(ctx context.Context) (<-chan Key, error) {
```

Comment:
Sounds like at this point we are 90% of the way to implementing ipfs/go-datastore